statchatui/src/Compare/Compare.jsx

```diff
@@ -37,43 +37,39 @@ const Compare = ({ signedInUserData }) => {
     setIsLoading(true); //sets the state to true before the api call is displaying the stats
     let stats1;
     let stats2;
-    let validUser = true;
     try {
       stats1 = await fetchStats(user1);
       stats2 = await fetchStats(user2);
-    } catch (error) {
```

Comment:
The previous location of the catch block was a bit better, since it wrapped around the piece that we expect to throw an error.